From cfaa222f9b91cb9885f57b1f965a453a7da03b88 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Wed, 15 Nov 2023 14:51:13 +0100
Subject: [PATCH] arm64: Kconfig: select STM32_EXTI on STM32 platforms

EXTI controller on STM32 platforms manages GPIO and wakeup interrupts.
Select STM32_EXTI has it is also a prerequisite for PINCTRL.

Change-Id: I45d49e9ade44d272579809ef31e1966f0d1aeb71
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -290,6 +290,7 @@ config ARCH_STM32
 	select GPIOLIB
 	select PINCTRL
 	select PINCTRL_STM32MP257
+	select STM32_EXTI
 	select ARM_SMC_MBOX
 	select ARM_SCMI_PROTOCOL
 	select COMMON_CLK_SCMI
